vllm/engine/arg_utils.py

```diff
@@ -39,7 +39,7 @@ class EngineArgs:
     quantization: Optional[str] = None
     enforce_eager: bool = False
     max_context_len_to_capture: int = 8192
-    disable_custom_all_reduce: bool = False
+    disable_custom_all_reduce: bool = True
```

Comment:
Can we keep this `False` by default? We'd like to minimize the number of knobs that users have to tune. Also, this fix does not allow our CLI users (e.g., for OpenAI server) to enable the custom all reduce.

Reply:
Sure